src/pow_fpga_accel.c

```diff
@@ -0,0 +1,162 @@
+/* 
+ * Copyright (C) 2018 dcurl Developers. 
+ * Copyright (c) 2018 Ievgen Korokyi. 
+ * Use of this source code is governed by MIT license that can be 
+ * found in the LICENSE file.
+ */
+
+#include <fcntl.h>
+#include <sys/mman.h>
+#include <unistd.h>
+#include "trinary.h"
+#include "constants.h"
+#include "pow_fpga_accel.h"
+
+#define HPS_TO_FPGA_BASE 0xC0000000
+#define HPS_TO_FPGA_SPAN 0x0020000
+#define HASH_CNT_REG_OFFSET 4
+#define TICK_CNT_LOW_REG_OFFSET 5
+#define TICK_CNT_HI_REG_OFFSET 6
+#define MWM_MASK_REG_OFFSET 3
+#define CPOW_BASE 0
+
+/* Set FPGA configuration for device files */
+#define CTRL_FPGA_POW "/dev/cpow-ctrl"
```

Comment:
Rename `CTRL_FPGA_POW` to `DEV_CTRL_FPGA` to represent the usage of device file(s).